### ISSUE TITLE
Thor for CLI

### ISSUE THREAD

**Issue by nullobject** [buildkite/buildbox-agent-ruby#7]
Have you considered using [thor](http://whatisthor.com/) for simplifying the CLI?

**Comment by keithpitt** [comment]
Good call. I wrote the CLI before I had a rule of "no dependencies" - but I decided to break that rule pretty quickly.

I'll take a look - thanks!
